compiler/one-cmds/onecc.template.cfg

```diff
@@ -1,29 +1,102 @@
+; # make `one-cmd` True and then uncomment its section
```

Comment:
(1) as this is not a md file nor that seems to be used to interpret ` code...
can you change this to single quotation ' ?

(2) we use `one-cmd` as the folder of this file but `one-cmd` is not exposed to the user who maybe using this file. so how about `To activate each step(or task), change to True for each [onecc] items and uncomment each corresponding section.` ?